SI/Sem.02/Pract.02/Task11.cpp

```diff
@@ -0,0 +1,56 @@
+﻿// Task11.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b, c;
+	cout << "Enter three numbers: ";
+	cin >> a >> b >> c;
+	int max, mid, min;
+	if (a>=b && a>=c)
```

Comment:
Your solution works but as similar to task 9 you can swap the numbers between them with three ifs and then print a*100 + b * 10 + c